libazureinit/src/user.rs

```diff
@@ -50,6 +50,29 @@ pub(crate) fn provision_ssh(
     Ok(())
 }
 
+pub fn add_user_for_passwordless_sudo(username: &str) -> Result<(), Error> {
+    // Create a file under /etc/sudoers.d with azure-init-user
+    let sudoers_path = "/etc/sudoers.d/azure-init-user";
+    let _ = File::create(sudoers_path)?;
```

Comment:
You can delete this line since the next line will also create the file.

Reply:
Done